src/main/java/org/folio/inventory/dataimport/handlers/actions/AbstractAuthorityEventHandler.java

```diff
@@ -180,19 +182,19 @@ protected void prepareEvent(DataImportEventPayload payload) {
     payload.getContext().put(targetRecordType().value(), new JsonObject().encode());
   }
 
-  private String getChunkIdHeader(DataImportEventPayload payload) {
+  protected String getChunkIdHeader(DataImportEventPayload payload) {
     return payload.getContext() == null ? "-" : payload.getContext().get(CHUNK_ID_HEADER);
   }
 
-  private String getRecordIdHeader(DataImportEventPayload payload) {
+  protected String getRecordIdHeader(DataImportEventPayload payload) {
```

Comment:
Do we need the change of methods visibility?

Reply:
I use this methods in `CreateAuthorityEventHandler`.
![image](https://user-images.githubusercontent.com/88006020/148274614-68a4c609-fd26-4ec6-aef4-8c5ed75a9bb0.png)

I can implement deduplication in the `AbstractAuthorityEventHandler` but it will affect the `UpdateAuthorityEventHandler` and have to compare profileActions, like `ActionProfile.Action.CREATE == actionProfile()`